### PR TITLE
Updated the Third Party Notices for 8.0 project templates

### PR DESCRIPTION
**Pull request by mkArtakMSFT**

### CODE REVIEW

File: src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/.template.config/template.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
   "precedence": "9000",
   "identity": "Microsoft.Web.Blazor.Server.CSharp.7.0",
```
Comment (TanayParikh) [inline src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/.template.config/template.json:13]: Does this need to be bumped to 8.0 as well?

**Review by TanayParikh — APPROVED** [review]
LGTM, do we have a new (major) release checklist that we can add this to?

File: src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/.template.config/template.json (hunk 1 above)
Comment (wtgodbe) [reply src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/.template.config/template.json:13]: Yes, I'm planning on doing name/precedence updates in a separate PR

**Review by wtgodbe — APPROVED** [review]

**Comment by wtgodbe** [comment]
> LGTM, do we have a new (major) release checklist that we can add this to?

It's already in https://github.com/dotnet/aspnetcore/blob/main/docs/UpdatingMajorVersionAndTFM.md

File: src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/.template.config/template.json (hunk 1 above)
Comment (mkArtakMSFT) [reply src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/.template.config/template.json:13]: I'm not sure the packages are already available or not, so for now will not touch these.

**Pull request merged**